packages/interactivity-router/src/index.ts

```diff
@@ -209,16 +209,18 @@ const isValidEvent = ( event: MouseEvent ) =>
 // Variable to store the current navigation.
 let navigatingTo = '';
 
+const navigationTexts = {
+	loadedFromServer: false,
```

Comment:
It seems to me that `loadedFromServer` does does not belong inside of
`navigationTexts` because it's a global flag rather than a "text".

Reply:
Changed this.